api/src/queries.js

```diff
@@ -0,0 +1,25 @@
+const items = {
+  get: `{
+        query(func: has(code)) @filter(eq(code, $code)) {	
+          code
+          description
+        }
+      }
```

Comment:
I think there's a minor '}' surplus (of 1) here 🙀 . Which makes me also think it may be a good idea to add some mapper tests (though not necessarily as part of this PR)

Reply:
Maybe I'm going crazy, but looks like the right amount to me (just perhaps badly indented)?

Reply:
Oops, GitHub didn't show `line 8`, you're right :).

Reply:
Haha my bad, I should probably have added the comment to the last bracket, I didn't realise GitHub would just go and hide everything below the line I selected 😆

Reply:
e43743fbd56cce4b2c628e7cab40e05ce4b5c31f.